python/cudf/cudf/core/column/column.py

```diff
@@ -2109,13 +2109,13 @@ def as_column(
             )
         elif isinstance(
             arbitrary.dtype, pd.api.extensions.ExtensionDtype
-        ) and not isinstance(arbitrary, pd.arrays.PandasArray):
+        ) and not isinstance(arbitrary, NumpyExtensionArray):
```

Comment:
If we want to support the pandas 2.0.x Series, this will still need to be `pd.arrays.PandasArray`. Starting in 2.1 was when it was renamed to `NumpyExtensionArray`

Reply:
Yup, I think this is handled at the top of the file.

Reply:
Ah OK sounds good